src/components/NewCommentForm.tsx

```diff
@@ -75,25 +167,40 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={cn('textarea', { 'is-danger': isBodyError })}
+            value={comment.body}
+            onChange={(e) => handleChangeBody(e)}
```

Comment:
```suggestion
            onChange={handleChangeBody}
```
do you need to pass whole even object here? if yes then it can be shortened